.travis.yml

```diff
@@ -1,21 +1,23 @@
 sudo: required
 env:
     matrix:
-    - IMAGE=centos:centos6 COMPONENTS=udt,myproxy,ssh
-    - IMAGE=centos:centos7 COMPONENTS=udt,myproxy,ssh
+    - IMAGE=centos:centos6 TASK=tests COMPONENTS=udt,myproxy,ssh
+    - IMAGE=centos:centos7 TASK=tests COMPONENTS=udt,myproxy,ssh
 #   Fedora builds are implemented but are disabled until we decide we
 #   need them; Travis limits our number of concurrent jobs so we don't
 #   want more than are necessary.
-#    - IMAGE=fedora:25      COMPONENTS=udt,myproxy,ssh
-#    - IMAGE=fedora:26      COMPONENTS=udt,myproxy,ssh
-#    - IMAGE=fedora:27      COMPONENTS=udt,myproxy,ssh
-    - IMAGE=centos:centos6 COMPONENTS=gram5
-    - IMAGE=centos:centos7 COMPONENTS=gram5
+#    - IMAGE=fedora:25     TASK=tests COMPONENTS=udt,myproxy,ssh
+#    - IMAGE=fedora:26     TASK=tests COMPONENTS=udt,myproxy,ssh
+#    - IMAGE=fedora:27     TASK=tests COMPONENTS=udt,myproxy,ssh
+    - IMAGE=centos:centos6 TASK=tests COMPONENTS=gram5
+    - IMAGE=centos:centos7 TASK=tests COMPONENTS=gram5
 #   The following gram tests failed for all fedora versions:
 #       nonblocking-register-test.pl, register-callback-test.pl, register-test.pl
-#    - IMAGE=fedora:25      COMPONENTS=gram5
-#    - IMAGE=fedora:26      COMPONENTS=gram5
-#    - IMAGE=fedora:27      COMPONENTS=gram5
+#    - IMAGE=fedora:25      TASK=tests COMPONENTS=gram5
+#    - IMAGE=fedora:26      TASK=tests COMPONENTS=gram5
+#    - IMAGE=fedora:27      TASK=tests COMPONENTS=gram5
+    - IMAGE=centos:centos7 TASK=tarballs COMPONENTS=ssh,gram5
```

Comment:
If I'm reading this right, we don't do anything with the `COMPONENTS` for the `tarballs` task, right?

Reply:
We don't, but it does determine which RPMs get yum installed first